frontend/src/store/dashboard/actions.ts

```diff
@@ -29,7 +29,7 @@ const getCategories = createAsyncThunk<
   AsyncThunkConfig
 >(ActionType.GET_CATEGORIES, async (_, { extra }) => {
   const { categoriesApi } = extra;
-  const categoriesDto = await categoriesApi.getAll();
+  const categoriesDto = await categoriesApi.getAllWithCourses();
```

Comment:
```suggestion
  const categoriesDto = await categoriesApi.getAll();
```
we don't want to fetch categories with courses